gtests/net/mptcp/mp_prio/mp_prio_client_inbound.pkt

```diff
@@ -0,0 +1,44 @@
+--tolerance_usecs=400000
```

Comment:
why is the tolerance higher here? On purpose?

Reply:
it's high because the MPTCP retransmission time (see packet 41) is varying a lot over different runs. Not sure if it's intentional though.